index.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const pkg = require('./package.json')
+const Funnel = require('broccoli-funnel')
+
+const flatiron = require('broccoli-flatiron')
+const fs = require('fs')
+const mergeTrees = require('broccoli-merge-trees')
+const path = require('path')
+const svgstore = require('broccoli-svgstore')
+
+module.exports = {
+  name: pkg.name,
+
+  included: function (app) {
+    this._super.included(app)
+  },
+
+  treeForAddon: function (tree) {
+    // Flatten the svgs into js imports with inline svg using flatiron and merge the result into the addon tree
+    var svgPaths = []
+
+    var addonSvgRoot = path.join(this.root, 'public', 'svgs')
```

Comment:
I'll raise an issue for this, public was the wrong location to put the svgs since we end up including both the files and the flattened module